pkg/normalizedtranscriptions/sipa.json

```diff
@@ -1,10 +1,12 @@
 [
   {
-    "conditions": {"aspiration": "aspirated", "voicing": "voiced"},
+    "conditions": {"aspiration": "aspirated", "phonation": "voiced"},
     "modifications": {"aspiration": null, "breathiness": "breathy"}
+    "note": {"converts voiced aspirated segments to breathy-voiced voiced"}
   },
   {
-    "conditions": {"voicing": "voiceless", "breathiness": "breathy"},
-    "modifications": {"voicing": "voiced"}
+    "conditions": {"phonation": "voiceless", "breathiness": "breathy"},
+    "modifications": {"phonation": "voiced"}
```

Comment:
same here.

Reply:
In fact, I think the syntax highlighting of github even gives some info on this.